app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  def index
+    tickets = Ticket.all
+    render json: tickets
+  end
+
+  def show
+    ticket = Ticket.find(params[:id])
+    render json: ticket
+  end
+
+  def create
+    ticket = Ticket.new(ticket_params)
+    if Ticket.where.not(Ticket.screening.id = ticket.screening.id, Ticket.seat.name = ticket.seat.name)
```

Comment:
What did you want to do here? Just pass params or assign something?

Reply:
I wanted to check if there is a record in Ticket db that has the same screening id and seat name as the one Im trying to create in line 15 and if not let it pass